frontend/src/pages/Contracts/Contract/ContractDetails/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { useQuery } from "react-query";
+import { useParams } from "react-router-dom";
+import { GDXAccordion } from "../../../../components/GDXAccordion";
+import { apiAxios } from "../../../../utils";
+import { ContractDetailsSection } from "./ContractDetailsSection";
+import { InvoiceSection } from "./InvoiceSection";
+
+export const ContractDetails = () => {
+  const { contractId } = useParams();
+  const [userHasEditCapability, setEditCapability] = useState(false);
+
+  const getContract = async () => {
+    const contract = await apiAxios().get(`contracts/${contractId}`);
+    contract.data.data.notes = contract.data.data.notes ?? "";
+    contract.data.data.description = contract.data.data.description ?? "";
+    return contract.data;
+  };
+
+  /**
+   * @todo Define a good type. "Any" type temporarily permitted.
```

Comment:
i don't think just this adds the todo to the linter, we need to make sure all todo's get captured somehow.

Reply:
I haven't looked into it but I thought JSDoc had some export function that would spit out formatted HTML of all our doc comments so I thought it would pick these @todos up there but I could be totally wrong.